R/Design-methods.R

```diff
@@ -1063,6 +1089,9 @@ setMethod("simulate",
       ## the reasons for stopping
       stopReasons <- lapply(resultList, "[[", "stop")
 
+      ## for dual simulations as it would fail in summary otherwise (for dual simulations reporting is not implemented)
+      stop_report <- matrix(TRUE, nrow = nsim)
```

Comment:
No, because this matrix is not supposed to contain missing values: 
![image](https://github.com/Roche/crmPack/assets/98522410/15f582fe-9772-4983-858c-7f1e2971b869)